libminifi/test/unit/PropertyTests.cpp

```diff
@@ -22,6 +22,39 @@
 #include "../../include/core/Property.h"
 #include "utils/StringUtils.h"
 #include "../TestBase.h"
+namespace {
+enum class ParsingStatus { ParsingFail , ParsingSuccessful , ValuesMatch };
+enum class ConversionTestTarget { MS, NS };
+
+ParsingStatus checkTimeValue(const std::string &input, int64_t t1, core::TimeUnit t2) {
+  int64_t TimeVal;
+  core::TimeUnit unit;
+  bool parsing_succeeded = org::apache::nifi::minifi::core::Property::StringToTime(input, TimeVal, unit);
+  if (parsing_succeeded) {
+    if (TimeVal == t1 && unit == t2) {
+      return ParsingStatus::ValuesMatch;
+    } else {
+      return ParsingStatus::ParsingSuccessful;
+    }
+  } else {
+    return ParsingStatus::ParsingFail;
+  }
+}
+
+bool conversionTest(uint64_t number, core::TimeUnit unit, uint64_t check, ConversionTestTarget conversionUnit) {
+  uint64_t out;
+  bool returnStatus;
```

Comment:
Minor: I know these will be eventually filled by the parser function, but I would recommend giving an initial value for the primitive types.

Reply:
Yeah that is always a better method. Will keep that in mind :)